nautobot/docs/user-guide/administration/installation/nautobot.md

```diff
@@ -152,37 +161,59 @@ Before we install anything into the virtualenv, we want to make sure that Pip is
-```no-highlight
+```no-highlight title="Update Python Pip and Wheel"
 pip3 install --upgrade pip wheel
 ```
 
+??? example "Example pip update output"
+
+    ```no-highlight
+    Requirement already satisfied: pip in ./lib/python3.10/site-packages (22.0.2)
+    Collecting pip
+    Downloading pip-24.0-py3-none-any.whl (2.1 MB)
+        ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.1/2.1 MB 14.8 MB/s eta 0:00:00
+    Collecting wheel
+    Downloading wheel-0.43.0-py3-none-any.whl (65 kB)
+        ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 65.8/65.8 KB 11.8 MB/s eta 0:00:00
+    Installing collected packages: wheel, pip
+    Attempting uninstall: pip
+        Found existing installation: pip 22.0.2
+        Uninstalling pip-22.0.2:
+        Successfully uninstalled pip-22.0.2
+    Successfully installed pip-24.0 wheel-0.43.0
+    ```
+
 By default, Pip will now install Python packages as wheels. In most cases this is desirable, however in some cases the wheel versions of packages may have been compiled with options differing from what is needed for a specific scenario. One such case presents itself here - the wheel for `pyuwsgi`, a key web server component of Nautobot, is built without SSL (HTTPS) support. This may be fine for a non-production deployment of Nautobot, such as in your lab, but for production deployments, not supporting HTTPS will not do at all. Fortunately, you can tell Pip when you don't want to use wheels for a specific package by passing the `--no-binary=<package>` CLI parameter. We'll use that below.
 
+Great! We have `NAUTOBOT_ROOT` ready for use by the `nautobot` user, so let's proceed to verifying the installation.
+
 ## Install Nautobot
 
-=== "PostgreSQL (Ubuntu and RHEL Flavors)"
+=== "PostgreSQL"
 
     Use Pip to install Nautobot:
 
-    ```no-highlight
+    ```no-highlight title="Pip install Nautobot"
     pip3 install --no-binary=pyuwsgi nautobot
     ```
 
-=== "MySQL (Ubuntu and RHEL Flavors)"
+=== "MySQL"
 
     Use Pip to install Nautobot with the MySQL client:
 
-    ```no-highlight
+    ```no-highlight title="Pip install Nautobot"
     pip3 install --no-binary=pyuwsgi "nautobot[mysql]"
     ```
 
-Great! We have `NAUTOBOT_ROOT` ready for use by the `nautobot` user, so let's proceed to verifying the installation.
+    !!! note "MySQL Unicode Settings"
+
+        If you are using MySQL as your database backend, and you want to enable support for Unicode emojis, please make sure to add `"OPTIONS": {"charset": "utf8mb4"}` to your `DATABASES` setting (upcoming). Please see the [configuration guide on MySQL Unicode settings](../configuration/required-settings.md#mysql-unicode-settings) for more information.
 
 ## Verify your Nautobot Installation
 
 You should now have a fancy `nautobot-server` command in your environment. This will be your gateway to all things Nautobot! Run it to confirm the installed version of `nautobot`:
 
-```no-highlight
+```no-highlight title="Verify Nautobot install"
 nautobot-server --version
 ```
```

Comment:
No example output here?

Reply:
Good call out.

Do we want to get into needing to update that periodically? I will research if there is a way to read the version of a Python project.